### PR TITLE
Touchup/travis migrate

### PR DESCRIPTION
**Pull request by mvxt**
# Description
Small updates to the Travis CI migration docs to account for newer features and clarify some details.

# Reasons
Was recently doing more competitive analysis work and thought I'd update this doc while I'm at it

### CODE REVIEW

File: jekyll/_cci2/migrating-from-travis.md [hunk 1]
```diff
@@ -8,56 +8,47 @@ description: "Migrating from Travis CI"
...
-- Enable the safe use of environment variables.
-- On each build, upload a `test-results.xml` file to be made accessible online.
+on this [example JavaScript repository](https://github.com/CircleCI-Public/circleci-demo-javascript-express/blob/master/.circleci/config.yml).
 
 ## Prerequisites
 
-This document assumes that you have an account with CircleCI that is linked
-to a repository. If you don't, consider going over our [getting started guide]({{ site.baseurl }}/2.0/getting-started/).
+This document assumes that:
+1. You have an account with CircleCI that is linked to a repository. If you don't yet,
```
Comment (teesloane) [inline jekyll/_cci2/migrating-from-travis.md:16]:
```suggestion
1. You have an account with CircleCI that is linked to a repository. If you do not have an account,
```

File: jekyll/_cci2/migrating-from-travis.md [hunk 2]
```diff
@@ -8,56 +8,47 @@ description: "Migrating from Travis CI"
...
+consider reading our [Getting Started Guide]({{ site.baseurl }}/2.0/getting-started/).
+1. You understand the [Basic Concepts]({{ site.baseurl }}/2.0/concepts/) in CircleCI.
+
+## Why Migrate to CircleCI?
+
+- **Scaling Concurrency**: You can run up to 80 concurrent jobs on our monthly Performance Plan or even more on an [annual plan](https://circleci.com/pricing/). Travis CI has capped concurrencies of 1, 2, 5, and 10 on each of their plans.
```
Comment (teesloane) [inline jekyll/_cci2/migrating-from-travis.md:22]:
```suggestion
- **Scaling Concurrency**: You can run up to 80 concurrent jobs on our monthly Performance Plan or even more on a [custom plan](https://circleci.com/pricing/). Travis CI has capped concurrencies of 1, 2, 5, and 10 on each of their plans.
```
I would refer to our "annual" plan as "custom" as that is the name of the plan (that happens to have a customizable annual billing cycle))

File: jekyll/_cci2/migrating-from-travis.md [hunk 3]
```diff
@@ -8,56 +8,47 @@ description: "Migrating from Travis CI"
...
-| deploy:           | [deploy:](https://circleci.com/docs/2.0/configuration-reference/#deploy)           | Use the deploy: step to deploy build artifacts                                                       |
+| language:,<br>os: | [docker, machine, macos, windows]({{ site.baseurl }}/2.0/executor-types/)          | CircleCI doesn't assume dependencies or commands based on a language. Instead, CircleCI allows you to use any of the major operating systems or [Docker images]({{ site.baseurl }}/2.0/circleci-images/#next-gen-language-images) as executors. Specify a Docker image with your needed prerequisites or use a VM and install other needed libraries (see pre-installed software for [Windows]({{ site.baseurl }}/2.0/hello-world-windows/), [macOS]({{ site.baseurl }}/2.0/testing-ios/#supported-xcode-versions), and [Linux]({{ site.baseurl }}/2.0/docker-to-machine/#pre-installed-software) images). Then, use `run:` steps as shown below to execute commands you require (e.g., install, build, test). |
```
Comment (teesloane) [inline jekyll/_cci2/migrating-from-travis.md:37]:
```suggestion
| language:,<br>os: | [docker, machine, macos, windows]({{ site.baseurl }}/2.0/executor-types/)          | CircleCI doesn't assume dependencies or commands based on a language; instead, choose an executor and use `run:` steps as shown below to execute commands you require (e.g., install, build, test). |
```

I think this line should be simplified. The information about pre-installed software and the links to other documents should all be found in the already linked to document on executor-types (which can be done in another PR). We should aim to keep table elements from getting too text-heavy (both for contributors to the docs and readers).

File: jekyll/_cci2/migrating-from-travis.md [hunk 4]
```diff
@@ -77,13 +68,13 @@ node_js:
 cache: npm
 ```
 
-For basic builds, a TravisCI configuration will leverage a language's best known
+For basic builds, a Travis CI configuration will leverage a language's best known
```
Comment (teesloane) [inline jekyll/_cci2/migrating-from-travis.md:71]: Good catch.

**Review by teesloane — CHANGES_REQUESTED** [review]
Nice work! I put in a couple suggestions / comments.

**Comment by mvxt** [comment]
@teesloane Committed the changes!

**Review by teesloane — APPROVED** [review]

**Comment by teesloane** [comment]
Lovely, approved and merged - thank you!

**Pull request merged**